web/src/components/core/PasswordInput.test.tsx

```diff
@@ -79,4 +94,87 @@ describe("PasswordInput Component", () => {
 
     screen.getByText("Password value updated!");
   });
+
+  it("renders keyboard reminders by default", async () => {
+    const { user } = installerRender(
+      <PasswordInput id="password" name="password" aria-label="User password" />,
+      {
+        withL10n: true,
+      },
+    );
+
+    screen.getByLabelText("User password");
+    screen.getByText(/^Using/);
+    screen.getByText("us");
+    screen.getByText(/keyboard$/);
+    await user.keyboard("{CapsLock}");
+    screen.getByText(/^CAPS LOCK/);
+    screen.getByText(/is on$/);
+    await user.keyboard("{CapsLock}");
+    expect(screen.queryByText(/^CAPS LOCK/)).toBeNull();
+  });
+
+  it("allow disabling reminders via reminders prop", async () => {
+    const { user } = installerRender(
+      <PasswordInput id="password" name="password" aria-label="User password" reminders={[]} />,
+      {
+        withL10n: true,
+      },
+    );
+
+    expect(screen.queryByText(/^Using/)).toBeNull();
+    expect(screen.queryByText(/^CAPS LOCK/)).toBeNull();
+    await user.keyboard("{CapsLock}");
+    expect(screen.queryByText(/^CAPS LOCK/)).toBeNull();
+  });
+
+  it("allows pikcing only the keymap reminder", async () => {
```

Comment:
typo: picking

Reply:
Didn't we have a check for spelling?

Reply:
> We had. But was deactivated for long time and I drop it.

See https://github.com/agama-project/agama/pull/2158

> I 've, however, an experimental approach in my development setup to check spelling on commit message

Which allows not polluting files with cspell directive comments